### ISSUE TITLE
Issues with displaying UI elements after resizing window.

### ISSUE THREAD

**Issue by krzychhoo** [BLCM/OpenBLCMM#2]
I have noticed an issue with properly displaying this search and help UI elements while resizing the window.
![obraz](https://user-images.githubusercontent.com/130290978/230772609-37834983-9356-475f-95a9-3041f949abd4.png)
I am using Arch Linux with GNOME 43.4 on Xorg with an Nvidia GPU. Video demonstrating the issue:

https://user-images.githubusercontent.com/130290978/230772914-2b561ce0-85d6-46a1-b4cf-a2561806b348.mp4

**Comment by apocalyptech** [comment]
Just for reference for myself, when I look into this more, I expect the crux of the matter is inside MainGUI's `addSearchLayer()`, with this:

```java
        jLayeredPane1.addComponentListener(new ComponentAdapter() {
            @Override
            public void componentResized(ComponentEvent e) {
                layer2.setBounds(jLayeredPane1.getBounds());
            }

            @Override
            public void componentMoved(ComponentEvent e) {
                layer2.setBounds(jLayeredPane1.getBounds());
            }
        });
        jScrollPane1.getViewport().addChangeListener((ChangeEvent e) -> {
            layer2.repaint();
        });
```

So it's theoretically updating its bounds whenever the parent layer is resized, and then a change listener queues up the repaint.  But clearly in some cases that stuff's not firing at the right times.  I expect that maybe the repaint is happening before the bounds have a chance to get updated, or something.

**Comment by apocalyptech** [comment]
Heh, well, I did spentd a fair amount of time on this today, and I honestly have no idea currently how to fix it up, and the time spent trying to figure it out doesn't quite seem worth the severity of the bug.  Will be leaving this bug open but I think it's likely to remain unfixed for the forseeable future.  Sorry!

**Comment by apocalyptech** [comment]
Heh, well, I left a note that I was just giving up on it, but in the process of putting together the smaller testcase I made for it, I figured I should probably check #java on Libera to see if anyone else could shed some light.  And, lo and behold, they got it figured out.  So, this should be good to go for the next version now!

(No idea when v1.4.0 will be out, but this'll be among the fixes once it does.)
